### PR TITLE
metadata: bump ubuntu and debian support to latest version (refs: #36)

### LINKED ISSUE #36: check debian support
this module once listed debian 7 in the metadata.json. We removed it because it's EOL. We need to check if recent debian versions work and add them back.

### PR DESCRIPTION
**Pull request by sbadia**
#### This Pull Request (PR) fixes the following issues

Fixes #36, when https://github.com/voxpupuli/puppet-mumble/pull/53 is merged.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -22,13 +22,13 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "14.04"
+        "18.04"
```
Comment (bastelfreak) [inline metadata.json]: do you know if Ubuntu 16.04 / Debian 9 work as well? If so, please also add them.

Comment (sbadia) [reply metadata.json]: Hi Tim, you're right !
PR just updated :-)

File: .sync.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 .travis.yml:
   docker_sets:
-    - set: debian8-64
-    - set: ubuntu1404-64
+    - set: debian9-64
+    - set: debian10-64
+    - set: ubuntu1604-64
+    - set: ubuntu1804-64
```
[the remark this answers is not in the document]
Comment (sbadia) [reply .sync.yml]: Hi @Dan33l make sense, I just updated the PR in this direction, thanks !

File: .travis.yml [hunk 3]
```diff
@@ -25,19 +25,35 @@ matrix:
     env: PUPPET_VERSION="~> 5.0" CHECK=build DEPLOY_TO_FORGE=yes
   - rvm: 2.5.3
     bundler_args: --without development release
-    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=debian8-64 BEAKER_HYPERVISOR=docker CHECK=beaker
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=debian9-64 BEAKER_HYPERVISOR=docker CHECK=beaker
     services: docker
   - rvm: 2.5.3
     bundler_args: --without development release
-    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=ubuntu1404-64 BEAKER_HYPERVISOR=docker CHECK=beaker
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=debian10-64 BEAKER_HYPERVISOR=docker CHECK=beaker
```
Comment (Dan33l) [inline .travis.yml]: Thank you. I forgotten that `puppet-agent` for Debian10 with Puppet5 is not yet released. This is the reason of failing test. The landing date of released is this october month. So, we can reasonably consider  that next `module_sync` will be run with `puppet-agent` released.

So can you remove this line? Sorry for this omission.

**Pull request merged**